jest/setup/index.js

```diff
@@ -4,5 +4,8 @@ import 'jest-prop-type-error'
 import './match-media.js'
 import './react-dom.js'
 
+import MutationObserver from 'mutation-observer'
```

Comment:
Thank you!  Can't wait to get this in.

In thinking about this, do you think that we should be shipping this as a polyfill to prod with any component(s)?  Or just in test?

Reply:
I think just in test. I'm pretty sure that prod teams should be responsible for managing this themselves. I'm not sure it be great to have more than one running which is a possibility if they rolled their own or bundled a different package. We could add it to our docs as a recommendation though.